.env.development

```diff
@@ -0,0 +1 @@
+REACT_APP_GOOGLE_AUTH_URL=http://localhost:8003/auth/google
```

Comment:
@Gwillison415 nice, thanks for using the `.development` suffix.

Reply:
@Gwillison415 let's just have one URL: REACT_APP_SHEETS_API, and append endpoints to it in the app. This needs to be addressed in this PR